service/src/test/resources/ehr/request/fhir/output/expected-xml-with-observations-inside-report.xml

```diff
@@ -129,16 +129,49 @@ This is some random free text</text>
         <specimen typeCode="SPC">
             <specimenRole classCode="SPEC">
                 <id root="test-id-3"/>
-                <id root="2.16.840.1.113883.2.1.4.5.2" extension="G,03.0999008.K"/>
+                <id root="2.16.840.1.113883.2.1.4.5.2" extension="DUMMY"/>
                 <effectiveTime>
-                    <center value="20030109000000"/>
+                    <center value="20030221115000"/>
                 </effectiveTime>
                 <specimenSpecimenMaterial determinerCode="INSTANCE" classCode="MAT">
-                    <desc>Blood</desc>
```

Comment:
Looking at the JSON, I can see no reason why this XML would change.

The DiagnosticReport has one Specimen, one Filing Comment Observation, and one Test Result which points at the previous Specimen.

Therefore in my mind there should only be one Specimen output, and no need for a dummy.